src/OrchardCore/OrchardCore.Data.YesSql/Migration/DataMigrationManager.cs

```diff
@@ -17,6 +17,9 @@ namespace OrchardCore.Data.Migration
     /// </summary>
     public class DataMigrationManager : IDataMigrationManager
     {
+        const string _updateFromPrefix = "UpdateFrom";
+        const string _asyncSuffix = "Async";
```

Comment:
No need for the suffix to be define, all are agreed that `async` should be used

Reply:
Better to leave such a comment in the issue instead of the PR. I think we should be consistent, if we support Task<int> UpdateFrom1() and Task<int> UpdateFrom1Async()

Then we should do the same with Create.

Reply:
> Better to leave such a comment in the issue instead of the PR.

My comment related to the PR change instead of the issue

Reply:
Yes but we already support both names Async and no Async names. So we can't remove that as it would break people. The issue suggests supporting both names in create as well for consistency.

Reply:
I never said to delete `async`. My comment is about removing the added suffix, nothing else

Reply:
I don't think I understand the ask.

Reply:
Leave and let's merge thsi